<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MBData</class>
 <widget class="QMainWindow" name="MBData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Data</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons/pyModSlaveQt.qrc">
    <normaloff>:/img/Bus.png</normaloff>:/img/Bus.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabIOs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>481</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabCoils">
     <attribute name="icon">
      <iconset resource="icons/pyModSlaveQt.qrc">
       <normaloff>:/img/DO.png</normaloff>:/img/DO.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Coils</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>5</y>
        <width>452</width>
        <height>364</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pbResetDO">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Reset</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/img/refresh-16.png</normalon>
            </iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkSimCoils">
           <property name="text">
            <string>Sim</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tvCoilsData">
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>330</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabDiscreteInputs">
     <attribute name="icon">
      <iconset resource="icons/pyModSlaveQt.qrc">
       <normaloff>:/img/DI.png</normaloff>:/img/DI.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Discrete Inputs</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>5</y>
        <width>452</width>
        <height>364</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pbResetDI">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Reset</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/img/refresh-16.png</normalon>
            </iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkSimDisInputs">
           <property name="text">
            <string>Sim</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tvDiscreteInputsData">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>330</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabHoldingRegisters">
     <attribute name="icon">
      <iconset resource="icons/pyModSlaveQt.qrc">
       <normaloff>:/img/AO.png</normaloff>:/img/AO.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Holding Registers</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>5</y>
        <width>452</width>
        <height>364</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pbResetAO">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Reset</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/img/refresh-16.png</normalon>
            </iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkSimHoldRegs">
           <property name="text">
            <string>Sim</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbHoldRegsType">
           <property name="maxVisibleItems">
            <number>2</number>
           </property>
           <item>
            <property name="text">
             <string>Decimal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hex</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tvHoldingRegistersData">
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>330</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabInputRegisters">
     <attribute name="icon">
      <iconset resource="icons/pyModSlaveQt.qrc">
       <normaloff>:/img/AI.png</normaloff>:/img/AI.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Input Registers</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>5</y>
        <width>452</width>
        <height>364</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="pbResetAI">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Reset</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/img/refresh-16.png</normalon>
            </iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkSimInputRegs">
           <property name="text">
            <string>Sim</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbInputRegsType">
           <property name="maxVisibleItems">
            <number>2</number>
           </property>
           <item>
            <property name="text">
             <string>Decimal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hex</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tvInputRegistersData">
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>330</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons/pyModSlaveQt.qrc"/>
 </resources>
 <connections/>
</ui>
